Changelog

```diff
@@ -5,7 +5,20 @@ py-amqp is fork of amqplib used by Kombu containing additional features and impr
 The previous amqplib changelog is here:
 http://code.google.com/p/py-amqplib/source/browse/CHANGES
 
-.. _version-2.2.0
+.. _version-2.2.1:
```

Comment:
@thedrow: I removed your commit, but now:
```
This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests.
Conflicting files
Changelog
```

I vaguely recall that this is why that commit was in this PR to begin with: rebasing to resolve the merge conflict.